lib/ooxml_parser/xlsx_parser/xlsx_data/view_model/workbook/worksheet/table_part/extension_list/extension/conditional_formattings/conditional_formatting/conditional_formatting_rule.rb

```diff
@@ -0,0 +1,78 @@
+# frozen_string_literal: true
+
+require_relative 'conditional_formatting_rule/differential_formatting_record'
+module OoxmlParser
+  # Class for `cfRule` data
+  class ConditionalFormattingRule < OOXMLDocumentObject
+    # @return [Symbol] Type of rule
+    attr_reader :type
+    # @return [Integer] Specifies position on the list of rules
+    attr_reader :priority
+    # @return [String] ID of rule
+    attr_reader :id
+    # @return [Integer] index of format
+    attr_reader :format_index
+    # @return [Symbol] Specifies whether rules with lower priority should be applied over this rule
+    attr_reader :stop_if_true
+    # @return [Symbol] Relational operator in value rule
+    attr_reader :operator
+    # @return [Symbol] Specifies whether percent is used in top/bottom rule
+    attr_reader :percent
+    # @return [Integer] Number of items in top/bottom rule
+    attr_reader :rank
+    # @return [Integer] Number of standard deviations in above/below average rule
+    attr_reader :standard_deviation
+    # @return [String] text value in text rule
+    attr_reader :text
+    # @return [Array, Formula] Formulas to determine condition
+    attr_reader :formulas
+    # @return [DifferentialFormattingRecord] Format
+    attr_reader :format
+
+    def initialize(parent: nil)
+      @formulas = []
+      super
+    end
+
+    # Parse ConditionalFormattingRule data
+    # @param [Nokogiri::XML:Element] node with ConditionalFormattingRule data
+    # @return [ConditionalFormattingRule] value of ConditionalFormattingRule data
+    def parse(node)
+      node.attributes.each do |key, value|
+        case key
+        when 'type'
+          @type = value.value.to_sym
+        when 'priority'
+          @priority = value.value.to_i
+        when 'id'
+          @id = value.value.to_s
+        when 'dxfId'
+          @format_index = value.value.to_i
+          @format = root_object.style_sheet.differential_formatting_records[@format_index]
```

Comment:
I think it's better to not place `@format` in this `when` branch but make a method

```
def format
  root_object.style_sheet.differential_formatting_records[@format_index]
end
```